### PR TITLE
Modify README appearance

### PR DESCRIPTION
**Pull request by konifar**
## Overview

Just tiny changes of README doc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,7 @@
 
 ## Best practice
 
-By running in regression testing prior to release, you can release new features more safely and confidently.You can automate that process by running an Autify for Mobile regression test automatically when the features are merged into the master/develop branch.
-
+By running in regression testing prior to release, you can release new features more safely and confidently. You can automate that process by running an Autify for Mobile regression test automatically when the features are merged into the master/develop branch.
```
Comment (konifar) [inline README.md:10]: Added space after `confidently.`

File: README.md [hunk 2]
```diff
@@ -18,4 +17,4 @@ By running in regression testing prior to release, you can release new features
 ## License
 
 for-mobile-ci-bitrise-integration-example © [Autify Engineers](https://github.com/autifyhq). Released under the [MIT License](LICENSE).<br/>
-Authored and maintained by [Autify Engineers](https://github.com/autifyhq) with help from [contributors](https://github.com/autifyhq/for-mobile-ci-bitrise-integration-example/graphs/contributors)
+Authored and maintained by [Autify Engineers](https://github.com/autifyhq) with help from [contributors](https://github.com/autifyhq/for-mobile-ci-bitrise-integration-example/graphs/contributors).
```
Comment (konifar) [inline README.md:20]: Added dot on the end.

**Comment by sotayamashita** [comment]
@konifar 
Thank you for your suggestion!
You will be the first-time contributor to this repo from outside Autify 🎉

At the same time, we care about your contribution in terms of your rights.
Upon merging, do you agree to allow other users to use the updated code?

Thank you!

**Comment by konifar** [comment]
> Upon merging, do you agree to allow other users to use the updated code?

Sure! Thanks for quick review!

**Review by sotayamashita — APPROVED** [review]
LGTM :tada:

**Pull request merged**